man/systemd-nspawn.xml

```diff
@@ -1402,7 +1402,92 @@
 
         <listitem><para>Equivalent to <option>--console=pipe</option>.</para></listitem>
       </varlistentry>
+    </variablelist>
+
+    </refsect2><refsect2>
+    <title>Credentials</title>
+
+    <variablelist>
+      <varlistentry>
+        <term><option>--load-credential=</option><replaceable>ID</replaceable>:<replaceable>PATH</replaceable></term>
+        <term><option>--set-credential=</option><replaceable>ID</replaceable>:<replaceable>VALUE</replaceable></term>
+
+        <listitem><para>Pass a credential to the container. Credentials are
+        limited-size binary or textual objects that may be passed from the host
+        to the container payload. Their primary purpose is passing
+        cryptographic keys or certificates, user account information or
+        identity information from host to container. The data is accessible
+        from the container via the file system, at a read-only location that
+        (if possible and permitted) is backed by non-swappable memory. The data
+        is only accessible to the root user inside the container. If enabled
+        the location for the credential objects may be retrieved via the
+        <varname>$CREDENTIALS_DIRECTORY</varname> environment variable set for
```

Comment:
do we want to namespace this environment variable? e.g. `SD_CREDENTIALS_DIRECTORY` or something similar?

Reply:
Ignore this. I guess this is in line with `STATE_DIRECTORY,RUNTIME_DIRECTORY` and friends.